src/main/java/com/spotinst/sdkjava/model/SpotinstElastigroupService.java

```diff
@@ -53,7 +53,7 @@ public static ApiElastigroup createElastigroup(ApiElastigroup groupToCreate, Str
         String uri = String.format("%s/aws/ec2/group", apiEndpoint);
 
         // Send the request
-        RestResponse response = RestClient.sendPost(uri, body, headers, queryParams);
+        RestResponse response = RestClient.sendPut(uri, body, headers, queryParams);
```

Comment:
Why this is changed to put? Was it changed just for testing.

Reply:
yes, corrected.